rules.bolt

```diff
@@ -33,7 +33,10 @@ path /ui {
 
 path /projects {
     read() { true }
-    write() { isAdmin() }
+    
+    path /{projects} is Projects {
```

Comment:
Really? Where is Projects schema defined? It's not going to work itself by magic